pkcs10/src/attribute.rs

```diff
@@ -0,0 +1,310 @@
+//! Attribute-related definitions as defined in X.501 (and updated by RFC 5280).
```

Comment:
This module seems to overlap considerably with https://docs.rs/x509-cert/latest/x509_cert/attr/index.html